es-vue-base/src/lib-components/EsZipCodeForm.vue

```diff
@@ -23,7 +23,7 @@
                 :id="inputId"
                 v-model="zipCode"
                 :class="{
-                    [`mb-${stackBreak}0 mr-${stackBreak}25`]: stackBreak,
+                    [`mb-${stackBreak}0 mr-${stackBreak}25`]: stackUntil,
```

Comment:
Ah, the reason it happens specifically when `stackUntil = "xs"` is because `EsZipCodeForm` needs to set `mb-0` to remove the margin on all breakpoints, but this will not override the `mb-50` from `FormInput` because it's not any more specific than `mb-50` (as opposed to `mb-sm-0`.

I've come up with a solution; let me know if it makes sense to you. Basically, I created a `polite-mb-50` class which is the same as `mb-50`, but doesn't have the `!important`, which means it can be overridden by the `mb-0` set by the parent component.

Reply:
Ah ok - yes, I think that solution makes sense to work around this!